Smoothing code.cpp

```diff
@@ -31,6 +31,31 @@ struct MovingAverage {
 std::uint16_t get_reading() {
```

Comment:
this is a function.
the errors are because you have added two functions _inside_ the `get_reading()` function.

later comments will explain